### PR TITLE
NGO Apply Content Updates

### PR DESCRIPTION
**Pull request by rgychiu**
Added more content to nonprofit apply page FAQ section, specifically the one detailing what the development process is like and how we use the development fee. I'm still working on the nonprofit apply timeline as well as ideas for how to change the usage of forms that we currently have, but currently that's being blocked by our sync with PR. Another thing that needs to be included is the pitchbook link in the footer and on the nonprofit apply page, but that is also dependent on ER.

### CODE REVIEW

**Review by choiboy98 — DISMISSED** [review]
🥇

File: data/ngoApplyData.js [hunk 1]
```diff
@@ -104,6 +104,17 @@ export default {
...
+          <br/>
+          In most cases, projects and development will span one semester, after which
+          our team will transfer ownership of the repository and any additional resources 
+          created during the development process. More recently, however, some projects have 
+          spanned the course of an entire year in an effort to ensure quality and impact. 
+          The development process follows the same structure in these cases, where a 
+          fully functional prototype is completed after one semester. However, these projects 
+          continue by gathering feedback during the next semester and iterating on the 
+          prototype to ensure that we're creating the most impactful and useful product for our 
+          clients.
```
Comment (angadgarg25) [inline data/ngoApplyData.js]: We should mention that this is usually a decision we come to as a group. Not like something they pick on their own.

File: data/ngoApplyData.js [hunk 2]
```diff
@@ -115,7 +126,7 @@ export default {
     {
       question: "Is there a development fee?",
       answer:
-        "We typically charge a fee of $1,000 to our clients for any work that we do. However, this fee can be reduced on a need basis."
+        "We typically charge a fee of $1,000 per semester to our clients for any work that we do. As a student-led nonprofit building products for other nonprofits, we typically incur costs from using services like Amazon AWS or other third-party APIs as we build our products. We also aim to develop our members professionally and foster a collaborative, passionate, and dedicated community by offering tech talks, networking events, and organization-wide retreats. Thus, the development fee mainly goes towards technology costs, professional development, and community events. However, we understand that nonprofits may have different financial circumstances and would be more than happy to reduce the fee or make accommodations on a need basis."
```
Comment (angadgarg25) [inline data/ngoApplyData.js]: Lets not say "happy to reduce the fee", but be a little more stern here (and honest).

File: data/ngoApplyData.js (hunk 1 above)
Comment (angadgarg25) [reply data/ngoApplyData.js]: Also maybe just saying the two options as equal rather than saying `in most cases` for one, and recently for the other.

Comment (angadgarg25) [reply data/ngoApplyData.js]: replacing `in most cases` with `traditionally` would also be fine imo.

**Review by angadgarg25 — CHANGES_REQUESTED** [review]
small things, looks good as a whole

File: data/ngoApplyData.js (hunk 2 above)
Comment (rgychiu) [reply data/ngoApplyData.js]: not happy to reduce the fee

**Review by angadgarg25 — APPROVED** [review]
🐓

**Pull request merged**